### PR TITLE
i18n - pulling from tx

### PR DESCRIPTION
**Pull request by stejskalleos**

### CODE REVIEW

File: locale/action_names.rb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Autogenerated!
 _("Action with sub plans")
+_("Check for long running tasks")
+_("Deliver notifications about long running tasks")
```
Comment (ekohl) [inline locale/action_names.rb:4]: Same question here as in leapp: where do these tasks come from? It's best to update the translations with minimal plugins present.

Comment (stejskalleos) [reply locale/action_names.rb:4]: When I did the changes only plugin enabled was foreman_ansible, nothing else

Comment (ekohl) [reply locale/action_names.rb:4]: https://github.com/theforeman/foreman-tasks/commit/03ce25bcd6f64bee07ffef665db422a89ae09c5c added them to foreman-tasks so we're going to see this in every repo that uses foreman-tasks. No way around it.

**Review by ekohl — CHANGES_REQUESTED** [review]
It looks like Japanese went wrong in Transifex. I'm going to check my script where I synced it. Now I worry it may be wrong in multiple places.

**Comment by ekohl** [comment]
I updated the translations with a correct encoding (https://github.com/ekohl/transync/commit/e0fea33a5fcd7b16a988a225d2982f1952c6a3dc). Please pull again.

**Comment by stejskalleos** [comment]
@ekohl updated, hope it's fine now.

**Review by ekohl — APPROVED** [review]
Yep, thanks!

**Pull request merged**